### PR TITLE
Add community templates

### PR DESCRIPTION
**Pull request by zhimadaren**

### CODE REVIEW

**Comment by zhimadaren** [comment]
Read me preview：
image

File: community templates/A Community Template Sample/A Community Template Sample.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                    "cancelAfter": {}
+                },
+                "annotations": [],
+                "lastPublishTime": "2021-08-20T04:31:17Z"
+            },
+            "dependsOn": [
+                "[concat(variables('factoryId'), '/datasets/OracleTable1')]"
+            ]
+        },
+        {
```
Comment (fhljys) [inline community templates/A Community Template Sample/A Community Template Sample.json:82]: Suppose you only need to change readme file? Why has this file been updated?

File: community templates/README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+In addition to writing a template file directly, you can also get a template from a pipeline in [Azure Data Factory](https://adf.azure.com). 
+For details about pipelines please refer [Data Integration Pipelines](https://go.microsoft.com/fwlink/?linkid=2181927).  
+
+To create a template from a pipeline, please go to [Azure Data Factory](https://adf.azure.com) and excute the following steps:
+
+1. Navigate to your data factroy, in the left side menu choose "Author" to open the authoring blade.
+
+   ![Open authoring pane](images/open-authoring-pane01.png?raw=true)
+
+2. Set up your [code repository](https://go.microsoft.com/fwlink/?linkid=2181834) if a repository is not configured in your data factory or switch to your existing configured git repository beacuse creating a template from a pipeline is only enabled in git mode.
```
Comment (fhljys) [inline community templates/README.md:20]: Seems 7 has nothing to do with other steps. Can't we export it directly? 
For step 7, maybe we should ask them to go to their git repo, add the properties we need

File: community templates/A Community Template Sample/A Community Template Sample.json (hunk 1 above)
Comment (zhimadaren) [reply community templates/A Community Template Sample/A Community Template Sample.json:82]: It has not been merged to the main branch

File: community templates/README.md (hunk 2 above)
Comment (zhimadaren) [reply community templates/README.md]: Updated

**Review by fhljys — APPROVED** [review]

**Pull request merged**